requirements.txt

```diff
@@ -15,10 +16,11 @@ psycopg2==2.7.1
 pyparsing==2.2.0
 pyScss==1.3.5
 PyYAML==3.12
+redis==2.10.5
 requests==2.13.0
 scipy==0.19.0
 six==1.10.0
 SQLAlchemy==1.1.6
 webassets==0.12.1
 Werkzeug==0.12.1
-xlrd==1.0.0
\ No newline at end of file
+xlrd==1.0.0
```

Comment:
I'm not quite sure why this is here? I've been trying to play around and get rid of it, but it won't go away?